### PR TITLE
Blog : Istio at KubeCon NA 2023

### PR DESCRIPTION
**Pull request by kfaseela**

### CODE REVIEW

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Istio at KubeCon North America 2023"
+description: A quick recap of Istio at KubeCon North America, at the McCormick Place in Chicago.
+publishdate: 2023-11-16
+attribution: "Faseela K, for the Istio Day Programme Committee"
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+The open source and cloud native community gathered from 6th to 9th November in Chicago for the last KubeCon of 2023. The four-day conference, organized by
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:9]:
```suggestion
The open source and cloud native community gathered from the 6th to 9th November in Chicago for the last KubeCon of 2023. The four-day conference, organized by
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+the Cloud Native Computing Foundation, was "twice the fun" for Istio, as we grew from a half day event in Europe in April 2023 to a full day CNCF co-located
+"Istio Day" in North America. To add to the excitement, Istio Day North America marks our first event as a CNCF graduated project. With Istio Day North
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:11]:
```suggestion
Istio Day in North America. To add to the excitement, Istio Day North America marks our first event as a CNCF graduated project. With Istio Day North
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+America, it is a WRAP for the major Istio events for 2023.  [Istio Day Europe](/blog/2023/istio-at-kubecon-eu/) was held in April, and alongside our
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:12]:
```suggestion
America, it's a wrap for the major Istio events for 2023.  [Istio Day Europe](/blog/2023/istio-at-kubecon-eu/) was held in April, and alongside our
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+{{< image width="75%"
+    link="./ica.jpg"
+    caption="Istio Day NA 2023, Istio Certified Associate (ICA)"
+    >}}
+
+The keynote was followed by [an end user talk by Kush Trivedi and Khushboo Mittal](https://www.youtube.com/watch?v=Uk0k8uhdyaA) from DevRev about their usage of Istio ServiceMesh. We had our much awaited session on ["Architect Ambient for Scale"](https://www.youtube.com/watch?v=S39yo6ZJ4iM), from John Howard, which stirred some interesting discussions in the community. We also had an interesting talk showcasing the collaboration between Lilt and Intel about ["Scaling AI powered translation services using Istio"](https://www.youtube.com/watch?v=jFJyLbHros0). After this we stepped into another [end user talk from Intuit](https://www.youtube.com/watch?v=Xe38vEygOqk) where Karim Lakhani explained about Intuit’s modern SaaS platform deploying multiple cloud native projects including Istio. The audience were excited when Mitch Connors and Christian Hernandez did [a live demo of  upgrading Istio Ambient Mesh with Argo](https://www.youtube.com/watch?v=o71PJAqy4P8) on a live public site, with a publicly accessible availability monitor.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:40]:
```suggestion
The keynote was followed by [an end user talk by Kush Trivedi and Khushboo Mittal](https://www.youtube.com/watch?v=Uk0k8uhdyaA) from DevRev about their usage of Istio ServiceMesh. We had our much awaited session on ["Architect Ambient for Scale"](https://www.youtube.com/watch?v=S39yo6ZJ4iM), from John Howard, which stirred some interesting discussions in the community. We also had an interesting talk showcasing the collaboration between Lilt and Intel about ["Scaling AI powered translation services using Istio"](https://www.youtube.com/watch?v=jFJyLbHros0). After this we stepped into another [end user talk from Intuit](https://www.youtube.com/watch?v=Xe38vEygOqk) where Karim Lakhani explained about Intuit’s modern SaaS platform deploying multiple cloud native projects including Istio. The audience were excited when Mitch Connors and Christian Hernandez did [a live demo of upgrading Istio ambient mesh with Argo](https://www.youtube.com/watch?v=o71PJAqy4P8) on a live public site, with a publicly accessible availability monitor.
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+{{< image width="75%"
+    link="./istioday-session-1.jpg"
+    caption="Istio Day NA 2023, Jam packed sessions"
+    >}}
+
+The event witnessed more focus on security as we moved forward to subsequent talks where Jackie Elliot from Microsoft took a dig at ["Istio Identity"](https://www.youtube.com/watch?v=QjmUDNXyckQ),  followed by a lightning talk from Kush Mansing from Speedscale showing [the impacts of running services with arbitrary code](https://www.youtube.com/watch?v=G6Y9JLnej0o) on Istio. We also had another [lightning talk from Xiangfeng Zhu](https://www.youtube.com/watch?v=lHUXvtSWdtQ), who is a PhD student at  University of Washington, where he showcased a tool developed to analyze and predict the performance overheads of Istio.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:47]:
```suggestion
The event witnessed more focus on security as we moved forward to subsequent talks where Jackie Elliot from Microsoft took a dig at ["Istio Identity"](https://www.youtube.com/watch?v=QjmUDNXyckQ), followed by a lightning talk from Kush Mansing from Speedscale showing [the impacts of running services with arbitrary code](https://www.youtube.com/watch?v=G6Y9JLnej0o) on Istio. We also had another [lightning talk from Xiangfeng Zhu](https://www.youtube.com/watch?v=lHUXvtSWdtQ), who is a PhD student at the University of Washington, where he showcased a tool developed to analyze and predict the performance overhead of Istio.
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
...Kiali for better debugging of Istio use cases. Ekansh Gupta from Zeta, and Nirupama Singh from Reskill pitched in another [end user talk explaining the best practices while upgrading Istio](https://www.youtube.com/watch?v=dl0sESwwm9c) in their production deployments. Istio multi-cluster is always a hot topic, and Lukonde Mwila and Ovidiu from AWS nailed it with perfection in the talk ["Bridging trust between multi-cluster meshes"](https://www.youtube.com/watch?v=FIVmVIJlLVw). We also had [an interactive panel discussion with the Istio TOC Members](https://www.youtube.com/watch?v=PEUiL2BPXds), where a lot of questions came in from the audience, and the good attendance for the discussion was a testament to the continued popularity of Istio. Istio Day concluded with [a brilliant workshop on getting started with Ambient Mesh](https://www.youtube.com/watch?v=SyjBSM-3dOY) from Christian Posta and Jim Barton from Solo.io, which is the hot topic all the audience were looking forward to. The slides for all the sessions can be found in the [IstioDay NA 2023 schedule](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/#thank-you-for-attending).
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:49]:
```suggestion
The [talk from the Kiali maintainers](https://www.youtube.com/watch?v=MX-Sym2EkGI) Jay Shaughnessy and Nick Fox, was very interesting, as it demonstrated many advanced ways of using Kiali for better debugging of Istio use cases. Ekansh Gupta from Zeta, and Nirupama Singh from Reskill pitched in another [end user talk explaining the best practices while upgrading Istio](https://www.youtube.com/watch?v=dl0sESwwm9c) in their production deployments. Istio multi-cluster is always a hot topic, and Lukonde Mwila and Ovidiu from AWS nailed it with perfection in the talk ["Bridging trust between multi-cluster meshes"](https://www.youtube.com/watch?v=FIVmVIJlLVw). We also had [an interactive panel discussion with the Istio TOC Members](https://www.youtube.com/watch?v=PEUiL2BPXds), where a lot of questions came in from the audience, and the good attendance for the discussion was a testament to the continued popularity of Istio. Istio Day concluded with [a brilliant workshop on getting started with ambient mesh](https://www.youtube.com/watch?v=SyjBSM-3dOY) from Christian Posta and Jim Barton from Solo.io, which is the hot topic all of the audience were looking forward to. The slides for all the sessions can be found in the [IstioDay NA 2023 schedule](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/#thank-you-for-attending).
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+{{< image width="75%"
+    link="./istioday-session-2.jpg"
+    caption="Istio Day NA 2023, Jam packed sessions"
+    >}}
+
+Istio’s presence at the conference did not end with Istio Day. The first day keynote of KubeCon + CloudNativeCon started with a project update video from Mitch Connors. It was also a proud moment for us, when two of our steering committee members,Lin Sun and Faseela K, bagged the CNCF community ["Chop Wood Carry Water" award](https://www.cncf.io/announcements/2023/11/08/cloud-native-computing-foundation-announces-2023-community-awards-winners/), presented by Chris Aniszczyk, CTO CNCF, at the second day keynote.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:56]:
```suggestion
Istio’s presence at the conference did not end with Istio Day. The first day keynote of KubeCon + CloudNativeCon started with a project update video from Mitch Connors. It was also a proud moment for us, when two of our steering committee members, Lin Sun and Faseela K, bagged the CNCF community ["Chop Wood Carry Water" award](https://www.cncf.io/announcements/2023/11/08/cloud-native-computing-foundation-announces-2023-community-awards-winners/), presented by Chris Aniszczyk, CTO CNCF, at the second day keynote.
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+{{< image width="75%"
+    link="./chop-wood-carry-water.jpg"
+    caption="KubeCon NA 2023, Chop Wood Carry Water Winners"
+    >}}
+
+Some of our maintainers and contributors made it to the CNCF Fall 2023 Ambassadors list as well, Lin Sun, Mitch Connors and Faseela K, to name a few.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:63]:
```suggestion
Some of our maintainers and contributors made it to the CNCF Fall 2023 Ambassadors list as well, Lin Sun, Mitch Connors, and Faseela K, to name a few.
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+    >}}
+
+The maintainer track session at KubeCon for Istio presented by TOC members John Howard and Louis Ryan grabbed great attention as they talked about the current ongoing efforts and future roadmap of Istio. The talk, and the size of the audience, underlined why Istio continues to be the most popular service mesh in the industry.
+
+{{< image width="75%"
+    link="./maintainer-track.jpg"
+    caption="KubeCon NA 2023, Maintainer Track"
+    >}}
+
+The Contribfest session on "Istio Hands-on Development and Contribution Workshop" by Lin Sun, Eric Van Norman, Steven Landow and Faseela K was also well received. It was great to see so many people interested in contributing to Istio and pushing their first Istio Pull Request at the end of the workshop.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:77]:
```suggestion
The Contribfest session on "Istio Hands-on Development and Contribution Workshop" by Lin Sun, Eric Van Norman, Steven Landow, and Faseela K was also well received. It was great to see so many people interested in contributing to Istio and pushing their first Istio Pull Request at the end of the workshop.
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+The following sessions at KubeCon North America were based on Istio and almost all of them had very good content:
+* [Take It to the Edge: Creating a Globally Distributed Ingress with Istio & K8gb](https://sched.co/1R2o5/)
+* [Contribfest: Istio: Hands-on Development and Contribution Workshop](https://sched.co/1R2q7/)
+* [Istio: The Past, Present and Future of the Project and Community](https://sched.co/1R2tA)
+* [Under the Hood: Exploring Istio's Lock Contention and Its Impact on Expedia's Compute Platform](https://sched.co/1R2uV)
+* [Service Mesh Battle Scars: Technology, Timing, and Tradeoffs](https://sched.co/1R2ts)
+* [Untangling Your Service Mesh with Feature Gates](https://sched.co/1R2v6)
+* [Flavors of Certificates in Service Mesh: The Whys and Hows!](https://sched.co/1R2wC)
+
+Istio had a full time kiosk in the project pavilion at KubeCon + CloudNativeCon North America 2023 , with the majority of questions asked around the schedule for Ambient Mesh being production ready.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:100]:
```suggestion
Istio had a fulltime kiosk in the project pavilion at KubeCon + CloudNativeCon North America 2023, with the majority of questions asked around the schedule for ambient mesh being production ready.
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+
+{{< image width="75%"
+    link="./istio-booth-1.jpg"
+    caption="KubeCon NA 2023, Istio Kiosk, Interesting Discussions"
+    >}}
+
+We are glad that the major question which we had at the Istio kiosk in Europe about *CNCF graduation* is already answered, and we assure everyone that we are working on the Ambient Mesh promotion with the same level of seriousness. Many of our members and maintainers offered support at the booth, helping us answer all the questions from our users.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:107]:
```suggestion
We are glad that the major question which we had at the Istio kiosk in Europe about *CNCF graduation* is already answered, and we assure everyone that we are working on the ambient mesh promotion with the same level of seriousness. Many of our members and maintainers offered support at the booth, helping us answer all the questions from our users.
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+
+{{< image width="75%"
+    link="./istio-booth-2.jpg"
+    caption="KubeCon NA 2023, Our maintainers at the Istio Kiosk"
+    >}}
+
+Another highlight about the Istio kiosk this time was that we had our new Istio T-shirts sponsored by Microsoft, Solo.io, Tetrate and Stackgenie for everyone to grab.
```
Comment (dhawton) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:114]:
```suggestion
Another highlight about the Istio kiosk this time was that we had our new Istio T-shirts sponsored by Microsoft, Solo.io, Tetrate, and Stackgenie for everyone to grab.
```

**Review by dhawton — COMMENTED** [review]
Initial read

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 13]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Istio at KubeCon North America 2023"
+description: A quick recap of Istio at KubeCon North America, at the McCormick Place in Chicago.
+publishdate: 2023-11-16
+attribution: "Faseela K, for the Istio Day Programme Committee"
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+The open source and cloud native community gathered from the 6th to 9th November in Chicago for the last KubeCon of 2023. The four-day conference, organized by
```
Comment (ilrudie) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:9]:
```suggestion
The open source and cloud native community gathered from the 6th to 9th of November in Chicago for the last KubeCon of 2023. The four-day conference, organized by
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+    >}}
+
+Another highlight about the Istio kiosk this time was that we had our new Istio T-shirts sponsored by Microsoft, Solo.io, Tetrate, and Stackgenie for everyone to grab.
+
+{{< image width="75%"
+    link="./istio-t-shirts.jpg"
+    caption="KubeCon NA 2023, The cool new Istio T-shirts"
+    >}}
+
+We would like to express our heartfelt gratitude to our platinum sponsors Google Cloud, for supporting Istio Day North America! Last but not least, we would like to thank our Istio Day Program Committee members for all their hard work and support!
```
Comment (ilrudie) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:121]: Would it be possible to list the members here or link to a listing that already exists?

**Review by ilrudie — COMMENTED** [review]
Great write up. Thanks. Left a couple non-blocking suggestions.

File: content/en/blog/2023/istio-at-kubecon-na/index.md (hunk 13 above)
Comment (dhawton) [reply content/en/blog/2023/istio-at-kubecon-na/index.md:9]: Just caught this, too... `the 6th to the 9th of`.. I missed a the.

**Comment by craigbox** [comment]
Review underway...

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+    >}}
+
+Istio Day kicked off with an opening keynote from the Program Committee chairs, Faseela K and Zack Butcher. The opening keynote made sure to recognize the day-to-day efforts of our contributors, maintainers, release managers, and users, with some awards for our top contributors and community helpers. Rob Salmond and Andrea Ma were recognized for their selfless efforts to the Istio community, and the top 20 contributors in the last 6 months were also
+appreciated.
+
+{{< image width="75%"
+    link="./top-contributors-1.jpg"
+    caption="Istio Day NA 2023, Top Contributors"
+    >}}
+
```
[the remark this answers is not in the document]
Comment (kfaseela) [reply content/en/blog/2023/istio-at-kubecon-na/index.md:27]: He was not there at the venue at the time of opening keynote, this is what we clicked in our phone :) I can see the youtube video and check if I can get a better screenshot

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+Istio’s presence at the conference did not end with Istio Day. The first day keynote of KubeCon + CloudNativeCon started with a project update video from Mitch Connors. It was also a proud moment for us, when two of our steering committee members, Lin Sun and Faseela K, bagged the CNCF community ["Chop Wood Carry Water" award](https://www.cncf.io/announcements/2023/11/08/cloud-native-computing-foundation-announces-2023-community-awards-winners/), presented by Chris Aniszczyk, CTO CNCF, at the second day keynote.
+
+{{< image width="75%"
+    link="./chop-wood-carry-water.jpg"
+    caption="KubeCon NA 2023, Chop Wood Carry Water Winners"
+    >}}
+
+Some of our maintainers and contributors made it to the CNCF Fall 2023 Ambassadors list as well, Lin Sun, Mitch Connors, and Faseela K, to name a few.
```
[the remark this answers is not in the document]
Comment (kfaseela) [reply content/en/blog/2023/istio-at-kubecon-na/index.md:63]: This was the group pic taken during Ambassadors breakfast, this has whole group.. I do have another pic with only Istio people..felt the group pic is better..

File: content/en/blog/2023/istio-at-kubecon-na/index.md (hunk 13 above)
Comment (kfaseela) [reply content/en/blog/2023/istio-at-kubecon-na/index.md]: Done

File: content/en/blog/2023/istio-at-kubecon-na/ica.jpg
[the remark this answers is not in the document]
Comment (kfaseela) [reply content/en/blog/2023/istio-at-kubecon-na/ica.jpg:1]: Fixed

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 17]
```diff
@@ -0,0 +1,128 @@
...
+    link="./top-contributors-1.jpg"
+    caption="Istio Day NA 2023, Top Contributors"
+    >}}
+
+{{< image width="75%"
+    link="./top-contributors-2.jpg"
+    caption="Istio Day NA 2023, Thank You Community Helpers!"
+    >}}
+
+The opening keynote also announced the availability of [the Istio Certified Associate (ICA) exam](https://www.cncf.io/blog/2023/11/06/introducing-the-istio-certified-associate-ica-certification-for-microservices-management/) for enrollment starting November 6th. We were also glad to showcase [a small video](https://www.youtube.com/watch?v=c5baPkXZEMU) of many of our contributors, vendors and end-users congratulating us for the CNCF graduation!
```
[the remark this answers is not in the document]
Comment (kfaseela) [reply content/en/blog/2023/istio-at-kubecon-na/index.md]: Done

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 18]
```diff
@@ -0,0 +1,128 @@
...
+    >}}
+
+The following sessions at KubeCon North America were based on Istio and almost all of them had very good content:
+* [Take It to the Edge: Creating a Globally Distributed Ingress with Istio & K8gb](https://sched.co/1R2o5/)
+* [Contribfest: Istio: Hands-on Development and Contribution Workshop](https://sched.co/1R2q7/)
+* [Istio: The Past, Present and Future of the Project and Community](https://sched.co/1R2tA)
+* [Under the Hood: Exploring Istio's Lock Contention and Its Impact on Expedia's Compute Platform](https://sched.co/1R2uV)
+* [Service Mesh Battle Scars: Technology, Timing, and Tradeoffs](https://sched.co/1R2ts)
+* [Untangling Your Service Mesh with Feature Gates](https://sched.co/1R2v6)
+* [Flavors of Certificates in Service Mesh: The Whys and Hows!](https://sched.co/1R2wC)
```
[the remark this answers is not in the document]
Comment (kfaseela) [reply content/en/blog/2023/istio-at-kubecon-na/index.md]: Done

File: content/en/blog/2023/istio-at-kubecon-na/istio-day-paris.jpg
[the remark this answers is not in the document]
Comment (kfaseela) [reply content/en/blog/2023/istio-at-kubecon-na/istio-day-paris.jpg:1]: Done :D

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 19]
```diff
@@ -0,0 +1,131 @@
...
+{{< image width="75%"
+    link="./chop-wood-carry-water.jpg"
+    caption="KubeCon NA 2023, Chop Wood Carry Water Winners"
+    >}}
+
+Some of our maintainers and contributors made it to the CNCF Fall 2023 Ambassadors list as well, Lin Sun, Mitch Connors, and Faseela K, to name a few.
+
+{{< image width="75%"
+    link="./cncf-ambassadors.jpg"
+    caption="KubeCon NA 2023, Istio CNCF Ambassadors"
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:71]:
```suggestion
    caption="The CNCF Ambassador group photo. Many Istio maintainers are in this picture!"
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 20]
```diff
@@ -0,0 +1,131 @@
...
+    >}}
+
+[The KubeCon maintainer track session for Istio](https://sched.co/1R2tA), presented by TOC members John Howard and Louis Ryan,  grabbed great attention as they talked about the current ongoing efforts and future roadmap of Istio. The technologies described in the talk, and the resulting size of the audience, underlined why Istio continues to be the most popular service mesh in the industry.
+
+{{< image width="75%"
+    link="./maintainer-track.jpg"
+    caption="KubeCon NA 2023, Maintainer Track"
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:78]:
```suggestion
    alt="The Istio maintainer track session at KubeCon NA 2023"
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 21]
```diff
@@ -0,0 +1,131 @@
...
+    >}}
+
+[The Contribfest Hands-on Development and Contribution Workshop](https://sched.co/1R2q7/) by Lin Sun, Eric Van Norman, Steven Landow, and Faseela K was also well received. It was great to see so many people interested in contributing to Istio and pushing their first pull request at the end of the workshop.
+
+{{< image width="75%"
+    link="./contrib-fest.jpg"
+    caption="KubeCon NA 2023, Contribfest Istio Workshop"
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:85]:
```suggestion
    alt="The Contribfest Istio Workshop at KubeCon NA 2023"
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 22]
```diff
@@ -0,0 +1,131 @@
...
+    >}}
+
+A much-awaited panel discussion on [Service Mesh Battle Scars: Technology, Timing and Tradeoffs](https://sched.co/1R2ts), led by the maintainers from three CNCF Service Mesh projects, had a huge crowd in attendance, and a lot of interesting discussions.
+
+{{< image width="75%"
+    link="./servicemesh-battle-scars-panel.jpg"
+    caption="KubeCon NA 2023, Service Mesh Battle Scars!"
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:92]:
```suggestion
    alt="The Service Mesh Battle Scars panel at KubeCon NA 2023"
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 23]
```diff
@@ -0,0 +1,131 @@
...
+    >}}
+
+Istio came up as a hot topic of discussion in several other KubeCon talks as well, here are a few we noticed:
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:95]:
```suggestion
Istio came up as a hot topic of discussion in several other KubeCon talks as well. Here are a few we noticed:

```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 24]
```diff
@@ -0,0 +1,131 @@
...
+    >}}
+
+We would like to express our heartfelt gratitude to our platinum sponsors Google Cloud, for supporting Istio Day North America! Last but not least, we would like to thank our Istio Day Program Committee members, for all their hard work and support!
+
+[See you in Paris in March 2024!](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/)
+
+{{< image width="75%"
+    link="./istio-day-paris.jpg"
+    caption="See you soon in Paris!"
+    >}}
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:131]:
```suggestion
{{< image width="75%"
    link="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/istio-day/"
    >}}
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 25]
```diff
@@ -0,0 +1,131 @@
...
+{{< image width="75%"
+    link="./istioday-session-2.jpg"
+    caption="Istio Day NA 2023, Kush Trivedi and Khushboo Mittal from DevRev on stage"
+    >}}
+
+Istio’s presence at the conference did not end with Istio Day. The first day keynote of KubeCon + CloudNativeCon started with a project update video from Mitch Connors. It was also a proud moment for us, when two of our contributors, Lin Sun and Faseela K, took home the prestigious CNCF community ["Chop Wood Carry Water" award](https://www.cncf.io/announcements/2023/11/08/cloud-native-computing-foundation-announces-2023-community-awards-winners/), presented by Chris Aniszczyk, CTO CNCF, at the second day keynote.
+
+{{< image width="75%"
+    link="./chop-wood-carry-water.jpg"
+    caption="KubeCon NA 2023, Chop Wood Carry Water Winners"
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:64]:
```suggestion
    caption="Chop Wood Carry Water winners, Faseela K and Lin Sun (second and third from left)"
```

File: content/en/blog/2023/istio-at-kubecon-na/index.md [hunk 26]
```diff
@@ -0,0 +1,131 @@
...
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+The open source and cloud native community gathered from the 6th to the 9th of November in Chicago for the final KubeCon of 2023. The four-day conference, organized by the Cloud Native Computing Foundation, was "twice the fun" for Istio, as we grew from a half-day event in Europe in April to a full day co-located event. To add to the excitement, Istio Day North America marked our first event as a CNCF graduated project.
+
+With Istio Day NA over, that's a wrap for our major community events for 2023. In case you missed them, [Istio Day Europe](/blog/2023/istio-at-kubecon-eu/) was held in April, and alongside our [Virtual IstioCon 2023](https://events.istio.io/) event, [IstioCon China 2023](/blog/2023/istiocon-china/) was held on September 26 in Shanghai, China.
+
+{{< image width="75%"
+    link="./welcome.jpg"
+    caption="Istio Day NA 2023, Welcome"
```
Comment (craigbox) [inline content/en/blog/2023/istio-at-kubecon-na/index.md:15]:
```suggestion
    alt="Istio Day NA 2023 welcome sign"
```

**Review by dhawton — APPROVED** [review]

**Comment by kfaseela** [comment]
lint fails due to https://github.com/istio/istio.io/issues/14155.. retrying

**Pull request merged**